merge/mergeRaw.go

```diff
@@ -120,7 +122,21 @@ func Combine(src *Merge, dst *Merge) *Merge {
 					d.Value = s.Value
 				}
 			}
+			override(d, src.conf)
 		}
 	}
 	return dst
 }
+
+// override the value if its key is in the conf.ConfigOverrides
+func override(d *Value, conf c.Conf) {
+	for k, v := range conf.ConfigOverrides {
+		if strings.Contains(d.Key, k) {
+			var buffer bytes.Buffer
+			buffer.WriteString(string([]rune(d.Value)[0]))
+			buffer.WriteString(v)
+			d.Value = buffer.String()
+			return
+		}
+	}
+}
```

Comment:
to handle each config you want to merge and overide. you can provide it a different ini file for that merge.